2021-07-27 07:51:48,347 INFO    MainThread:30496 [wandb_setup.py:_flush():69] setting env: {}
2021-07-27 07:51:48,347 INFO    MainThread:30496 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-27 07:51:48,347 INFO    MainThread:30496 [wandb_init.py:_log_setup():337] Logging user logs to /home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/wandb/run-20210727_075148-1uvpl49c/logs/debug.log
2021-07-27 07:51:48,347 INFO    MainThread:30496 [wandb_init.py:_log_setup():338] Logging internal logs to /home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/wandb/run-20210727_075148-1uvpl49c/logs/debug-internal.log
2021-07-27 07:51:48,347 INFO    MainThread:30496 [wandb_init.py:init():370] calling init triggers
2021-07-27 07:51:48,347 INFO    MainThread:30496 [wandb_init.py:init():375] wandb.init called with sweep_config: {}
config: {}
2021-07-27 07:51:48,348 INFO    MainThread:30496 [wandb_init.py:init():419] starting backend
2021-07-27 07:51:48,348 INFO    MainThread:30496 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-27 07:51:48,358 INFO    MainThread:30496 [backend.py:ensure_launched():135] starting backend process...
2021-07-27 07:51:48,365 INFO    MainThread:30496 [backend.py:ensure_launched():139] started backend process with pid: 30524
2021-07-27 07:51:48,375 INFO    MainThread:30496 [wandb_init.py:init():424] backend started and connected
2021-07-27 07:51:48,381 INFO    MainThread:30496 [wandb_init.py:init():472] updated telemetry
2021-07-27 07:51:48,395 INFO    MainThread:30496 [wandb_init.py:init():491] communicating current version
2021-07-27 07:51:49,195 INFO    MainThread:30496 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.11.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-27 07:51:49,195 INFO    MainThread:30496 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-27 07:51:49,483 INFO    MainThread:30496 [wandb_init.py:init():529] starting run threads in backend
2021-07-27 07:51:52,412 WARNING MainThread:30496 [wandb_init.py:init():777] interrupted
Traceback (most recent call last):
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 757, in init
    run = wi.init()
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 532, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 807, in communicate_run_start
    result = self._communicate(rec)
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 545, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
